core/storage/user/gae_models.py

```diff
@@ -2688,7 +2691,7 @@ def get_deletion_policy():
         """DeletedUserModel contains only hashes of usernames that were
         deleted.
```

Comment:
In addition to this, explain why it must be kept (i.e. to prevent reuse of the username, and subsequent user confusion).

Reply:
Done.